src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -114,7 +114,8 @@ public function storeOperatesOnASingleChannel($currencyCode = null)
      */
     public function theStoreOperatesOnAChannelNamed($channelIdentifier, $currencyCode = null)
     {
-        $defaultData = $this->defaultChannelFactory->create($channelIdentifier, $channelIdentifier, $currencyCode);
+        $channelCode = $this->resolveChannelCode($channelIdentifier);
```

Comment:
We can use `StringInflector::nameToLowercaseCode(string $value)` method instead.